fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/db2/Db2Adapter.java

```diff
@@ -365,43 +365,83 @@ public void detachPartition(String schemaName, String tableName, String partitio
         final String qname = DataDefinitionUtil.getQualifiedName(schemaName, tableName);
         final String detachedName = DataDefinitionUtil.getQualifiedName(schemaName, intoTableName);
         final String ddl = "ALTER TABLE " + qname + " DETACH PARTITION " + partitionName + " INTO " + detachedName;
-        runStatement(ddl);
+        
+        try {
+            runStatement(ddl);
+        }
+        catch (DataAccessException x) {
```

Comment:
format, how about moving "catch (DataAccessException x) {" to above line?

Reply:
done